### PR TITLE
Open Loki and PromeCIeus in new windows

### PR DESCRIPTION
**Pull request by stbenjam**
Currently when clicking from Prow, it opens within the small frame.

blank

### CODE REVIEW

**Comment by stbenjam** [comment]
@vrutkovs Could you take a look? Thanks!

File: ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh [hunk 1]
```diff
@@ -517,6 +517,6 @@ echo "Promtail manifests created, the cluster can be found at https://grafana-lo
 if [[ -f "/usr/bin/python3" ]]; then
   ENCODED_INVOKER="$(python3 -c "import urllib.parse; print(urllib.parse.quote('${OPENSHIFT_INSTALL_INVOKER}'))")"
   cat >> ${SHARED_DIR}/custom-links.txt << EOF
-  <a href="https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%5B%22now-24h%22,%22now%22,%22Grafana%20Cloud%22,%7B%22expr%22:%22%7Binvoker%3D%5C%22${ENCODED_INVOKER}%5C%22%7D%20%7C%20unpack%22%7D%5D">Loki</a>
+  <a target="_blank" href="https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%5B%22now-24h%22,%22now%22,%22Grafana%20Cloud%22,%7B%22expr%22:%22%7Binvoker%3D%5C%22${ENCODED_INVOKER}%5C%22%7D%20%7C%20unpack%22%7D%5D">Loki</a>
```
Comment (vrutkovs) [inline ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh:520]: I'm not sure if that would work. SpyGlass renders these in an IFrame. HTML is :wastebasket: and `_blank` can't open links in a new tab from iframes.

Lets see if rehearsals would work

Comment (stbenjam) [reply ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh:520]: Yea you're right, it doesn't work.

What if we did `<base target="_parent">` in spyglass?

Comment (stbenjam) [reply ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh:520]: Hm, maybe that won't work either.  Maybe at least a note to control + click those links? 

It wasn't immediately obvious to me what was going on when I clicked the link and the iframe went what appeared to be blank.

Comment (stbenjam) [reply ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh:520]: According to https://www.w3schools.com/tags/att_iframe_sandbox.asp, we can add `allow-popups` to the sandbox, that should let us set the target.

What do you think of changing https://github.com/kubernetes/test-infra/blob/master/prow/spyglass/lenses/html/template.html#L16 ?

Comment (stbenjam) [reply ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh:520]: This works now!

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/20715/rehearse-20715-pull-ci-openshift-builder-release-4.9-e2e-aws-cgroupsv2/1420840431641432064

**Review by vrutkovs — APPROVED** [review]
/lgtm

**Pull request merged**